src/components/DisclosurePage/DisclosurePage.scss

```diff
@@ -0,0 +1 @@
+// Fill this later
```

Comment:
So if you ever wanted to use Bootstrap variables like `$spacer` in a component SCSS file like this, you will have to import the variables. [Like:](https://github.com/edx/studio-frontend/blob/master/src/components/AssetsTable/AssetsTable.scss#L1)

```scss
@import 'edx-bootstrap';
```

The contents of [that file](https://github.com/edx/studio-frontend/blob/827089472226fe2c108eecd363c70868b07dc41d/src/_edx-bootstrap.scss) tries to just be variable definitions, no actual styles. This is because Webpack ends up including the contents of `_edx-bootstrap.scss` in the final extracted studio-frontend scss repeatedly for every component scss that imports that. I think this is dumb, but haven't found a way to make Webpack dedupe it and only include it once. So I just kept it to variable definitions for now.

And since we are no longer including ALL of Bootstrap in every component's SCSS, we can no longer do things like:

```scss
.foo {
  @extend '.bar';
}
```

Because, `.bar` is a style that is only defined in the project's root SCSS.

I think we should do more investigation on how other projects solve this problem. I suspect we are doing something wrong.